### ISSUE TITLE
Does NOT support M40(maxwell) GPU any more? 

### ISSUE THREAD

**Issue by kisow** [NVIDIA/triton-inference-server#1246]
**Description**
(Excuse me. I have a very poor english...)
TRTIS can NOT load torchscript or tensorflow model in M40 GPU since container 19.08.
**Using 19.07 container, it works well.** But, in all containers from 19.08 to 20.03,  
it does NOT work with " no GPUs are available" and "failed to get energy consumption for GPU 0, NVML_ERROR 3" messages.
 
**TRTIS Information**
GPU : M40 (24GB)
Driver : 440.33.01
CUDA : 10.2
> What version of TRTIS are you using?

TRTIS version: 1.5.0 ~ 1.13.0
TRTIS container: 19.08 ~ 20.02, tritonserver container 20.03

> Are you using the TRTIS container or did you build it yourself?

I have used TRTIS official containers.

**To Reproduce**
I have converted to torchscript model in pytorch:19.10 container from nvidia and tried to load model by TRTIS in tensorrtserver:19.10 with M40. But, it fails to load with  **" no GPUs are available"** message. and "failed to get energy consumption for GPU 0, NVML_ERROR 3" message also printed again. (#101) 

I have also tried with 19.08 ~ 20.03 containers, but, all tries have failed with same error.
```
===============================
== TensorRT Inference Server ==
===============================

NVIDIA Release 19.10 (build 8266503)

Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.

Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
NVIDIA modifications are covered by the license terms that apply to the underlying
project or file.

I0331 11:00:40.270582 1 metrics.cc:160] found 1 GPUs supporting NVML metrics
I0331 11:00:40.276705 1 metrics.cc:169]   GPU 0: Tesla M40 24GB
I0331 11:00:40.277057 1 server.cc:110] Initializing TensorRT Inference Server
E0331 11:00:40.313092 1 model_repository_manager.cc:1453] instance group nsa-1.1_0 of model nsa-1.1 has kind KIND_GPU but no GPUs are available
E0331 11:00:40.313168 1 main.cc:1099] error: creating server: INTERNAL - failed to load all models
W0331 11:00:42.281723 1 metrics.cc:272] failed to get energy consumption for GPU 0, NVML_ERROR 3
```

**Expected behavior**
Plase, support M40 GPU~

**Comment by deadeyegoodwin** [comment]
Only GPUs with CUDA compute capability 6.0 or greater (that is pascal or later) are currently supported. This is due to what we can test and what the underlying ML frameworks that we depend on officially support. The reason you get the "no GPUs" message is that no 6.0+ GPU is available. But starting in 20.03 there is a command-line flag that you can use to avoid that check. See release announcement for the --min-supported-compute-capability flag. https://github.com/NVIDIA/triton-inference-server/releases/tag/v1.12.0. Look up your GPU's compute capability here: https://developer.nvidia.com/cuda-gpus#compute

The "failed to get energy consumption..." message is just a warning (the 'W' at the beginning of the line tells you that). It means that your GPU doesn't report that metric. If you don't care about metrics you can use the --allow-metrics=false flag to turn them off and then you won't see the warning.
